addons/mail/static/tests/message/message.test.js

```diff
@@ -1816,6 +1816,34 @@ test("Click on view reactions shows the reactions on the message", async () => {
     await contains(".o-mail-MessageReactionMenu", { text: "😅1" });
 });
 
+test("Reactions are in order by id", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "channel",
+        name: "channel1",
+    });
+    pyEnv["mail.message"].create({
+        body: "Hello world",
+        res_id: channelId,
+        message_type: "comment",
+        model: "discuss.channel",
+        reaction_ids: [
+            pyEnv["mail.message.reaction"].create({
+                content: "➕",
+                partner_id: serverState.partnerId,
+            }),
+            pyEnv["mail.message.reaction"].create({
+                content: "🔢",
+                partner_id: serverState.partnerId,
+            }),
+        ],
+    });
```

Comment:
We don't really care of technical id order. What's important is the order by date (older reactions come first).
It just happens this is the same with the current implementation.